server/config/default.yaml

```diff
@@ -175,4 +175,10 @@ roles:
       name: "Master"
       description: "All permissions"
       # Note that the ability to "manageScripts" means that one circumvent the "query" permission on a signal because a script has always access to all signals in the signal set.
-      permissions: ["view", "edit", "delete", "insertRecord", "editRecord", "deleteRecord", "query", "share", "reindex", "createSignal", "manageScripts"]
+      permissions: ["view", "edit", "delete", "insertRecord", "editRecord", "deleteRecord", "query", "share", "reindex", "createSignal", "manageScripts", "createAnomaly"]
+
+  anomaly:
```

Comment:
Not sure it is needed - anomaly detection is a product of jobs and owned signals, not necessarily an entity, just a relationship